SASS/style.scss

```diff
@@ -0,0 +1,629 @@
+*,
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+#offcanvasDarkNavbar{
+  background-color: #2cc9b4 !important;
+}
+body {
+  #banner {
+    background-image: url(../images/banner.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 100vh;
+    position: relative;
+    padding-top: 20px;
+ 
+   .navbar-toggler{
+    position: absolute;
+    right: 10%;
+   
+   }
+    border: none;
+    
+    #nav-button {
+      position: absolute;
+      top: 20%;
+      font-weight: 400;
+      border: 0;
+      right: 10%;
+    }
+.nav-item{
+  font-size: 20px;
+  border-radius: 20px 0px;
+  margin-bottom: 20px;
+}
+    #banner-row {
+      #banner-col {
+        text-align: center;
+        position: absolute;
+        left: 50%;
+        top: 50%;
+        transform: translate(-50%, -50%);
+
+        .banner-heading {
+          font-family: "Courier New", Courier, monospace;
+          letter-spacing: 1rem;
+          font-weight: lighter;
+          font-size: 3rem;
```

Comment:
for font-size and color please use variable

Reply:
used